ChangeLog.md

```diff
@@ -20,6 +20,12 @@ See docs/process.md for more on how version tagging works.
 
 3.1.32 (in development)
 -----------------------
+- In Wasm exception mode (`-fwasm-exceptions`), when
+  `EXCEPTION_STACK_TRACES` is enabled, uncaught exceptions will display stack
+  traces. Note that stack traces are enabled when either of `ASSERTIONS` or this
```

Comment:
How about "This defaults to true when `ASSERTIONS` are enabled".

Reply:
Done. Not sure if I should use 'are' or 'is' for `ASSERTIONS`, because it is an option name.. (Used 'is' for now. Let me know if that doesn't sound right)